src/Manticoresearch/ResultHit.php

```diff
@@ -31,6 +31,10 @@ public function getScore() {
 		return $this->data['_score'];
 	}
 
+	public function getKnnDist() {
+		return $this->data['_knn_dist'] ?? 0;
```

Comment:
Doesn't it make sense to return `false` or `null` instead of 0 if there's no `_knn_dist` to avoid the confusion with the real `0` (when the vectors are 100% identical)?

Reply:
Sure, but I think we should stick to the same method we use in similar situations in the library. @Nick-S-2018, what's your take on the best approach here?